src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Option.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import optionStyles from './option.scss';
+
+const SELECTED_ICON = 'check';
+
+export default class Option extends React.PureComponent {
+    props: {
+        selected: boolean,
+        disabled: boolean,
+        focus: boolean,
+        value: string,
+        children?: string,
+        onClick?: (s: string) => void,
+    };
+
+    static defaultProps = {
+        disabled: false,
+        selected: false,
+        focus: false,
+        value: '',
+    };
+
+    item: HTMLElement;
+    button: HTMLButtonElement;
+
+    componentDidMount() {
+        if (this.props.focus) {
```

Comment:
Imagine you have 10 selects on one page which all have a selected element. If at this point one would set the window focus on the button if the option is selected, the element which has to focus would be more or less random. Moreover the element which gets the focus is not always selected.